hrm-domain/hrm-core/profile/sql/profile-list-sql.ts

```diff
@@ -55,11 +55,14 @@ const selectProfilesUnorderedSql = (whereClause: string) => `
 `;
 
 const listProfilesPaginatedSql = (whereClause: string, orderByClause: string) => `
-  ${getProfilesSqlBase(`
+  ${getProfilesSqlBase(
+    `
     ${selectProfilesUnorderedSql(whereClause)}
     ${orderByClause}
     OFFSET $<offset>
-    LIMIT $<limit>`)}
+    LIMIT $<limit>`,
+    false,
```

Comment:
Can we start naming the parameters rather than using positional ones? Of course, not in this PR, but as a note in general.